tests/test_meltingpot.py

```diff
@@ -0,0 +1,57 @@
+"""Tests the functionality of the MeltingPotCompatibility wrapper on meltingpot substrates."""
+
+import numpy as np
+import pytest
+
+from shimmy.meltingpot_compatibility import MeltingPotCompatibilityV0
+from meltingpot.python.configs.substrates import SUBSTRATES
+from pettingzoo.test import parallel_api_test
+
+
+@pytest.mark.parametrize("substrate_name", SUBSTRATES)
+def test_passing_substrates(substrate_name):
+    """Tests the conversion of all openspiel envs."""
+    for _ in range(5):
```

Comment:
Why are we running this 5 times?

Reply:
I just mirrored the functionality of the openspiel tests, but maybe both should be removed as it does seem redundant and slow.

Reply:
@jjshoots Do you know why?

Reply:
On Openspiel this was needed because some of the envs weren't deterministic, and so randomizing it with 5 seeds can help reveal problems that only pop up sometimes.

Reply:
Ok, so we shouldn't have that problem in meltingpot